manifests/dev/vsphere-7.0u2/supervisorcluster/1.18/vsphere-csi-controller.yaml

```diff
@@ -93,20 +83,30 @@ roleRef:
   kind: ClusterRole
   name: wcp-privileged-psp
 subjects:
-# For the kube-system nodes.
-- apiGroup: rbac.authorization.k8s.io
-  kind: Group
-  name: system:serviceaccounts:vmware-system-csi
+  # For the kube-system nodes.
```

Comment:
`For vmware-system-csi nodes`
This applies to all yamls

Reply:
Done